packages/ui/lib/components/Form/FieldLayout.tsx

```diff
@@ -7,6 +7,8 @@ export interface Props {
   size?: Size
   error?: string
   success?: string
+  required?: boolean
+  optional?: boolean
```

Comment:
```suggestion
  required?: boolean
  optional?: boolean
```

i think we can just use `required`, IMO is kinda confusing

Reply:
No it is important to have things be explicit. It would be pretty bad to just add `optional` to every single input if we don't want it.